### DISCUSSION TITLE
Unable to add `headers` to vercel.json, deployments unable to build

### DISCUSSION THREAD

**Discussion by gabbibrooks** [vercel/community#2883] [Help] [answered]
### Summary

Hello,

I have been attempting to add a headers block to my vercel.json config for my react-spa app and I have been unable to do so. I followed the documentation but no matter what I try, vercel does not allow me to create a deployment with a `headers` block in my configuration.
This is how my file looks:
```json
{
  "rewrites": [{ "source": "/(.*)", "destination": "/index.html" }],
  "headers:": [
    {
      "source": "/manifest.webmanifest",
      "headers": [
        {
          "key": "Content-Type",
          "value": "application/manifest+json"
        }
      ]
    },
    {
      "source": "/assets/(.*)",
      "headers": [
        {
          "key": "Cache-Control",
          "value": "public, max-age=31536000, immutable"
        }
      ]
    }
  ]
}
```

But with this configuration I receive the following error from vercel when pushing to github and trying to create a pr.

Deployment failed with the following error:
~~~
should NOT have additional property `headers:`
~~~

Any idea why I cannot add a headers block?

### Steps to Reproduce

Add a headers array to `vercel.json` and try to create a new deployment.

Deployments are not even allowed to be created

**Comment by smaeda-ks** [comment, marked as answer]
@brooksztb you had a typo `"headers:": [` -> `"headers": [`

**Reply by gabbibrooks** [reply]
Oh wow, thank you so much. I cannot believe I missed that.
